_includes/scenarios/integrate/request-and-response-introduction.md

```diff
@@ -0,0 +1,50 @@
+You want to share Attributes with a contact or receive Attributes from them?
+You need an authentication or a consent from a peer?
+You wish to make edits to a Relationship or ensure the correctness of data?
+All of these endeavors and more can be achieved by the means of Requests.
+They are the main instrument when it comes to interacting with other Identities going beyond the simple exchange of messages.
```

Comment:
I think you write about enmshed Messages here and not messages in general. In that case please write `Messages` - this is how we make sure in the whole docs that the reader knows that we talk about enmeshed terms.